lib/slide_hub/convert_util.rb

```diff
@@ -52,10 +52,13 @@ def pdf_to_transcript(dir, file)
       page_count.times do |i|
         current_page = i + 1
         cmd = "cd #{dir} && pdftotext #{file} -f #{current_page} -l #{current_page} - > #{dir}/#{current_page}.txt"
-        system(cmd)
-
-        str = File.read("#{dir}/#{current_page}.txt")
-        str.gsub!(/([\r|\n|\t| |　|\u{2028}]+)/, ' ')
+        result = system(cmd)
+        if result && File.exists?("#{dir}/#{current_page}.txt")
```

Comment:
__[RuboCop]__

Lint/DeprecatedClassMethods: `File.exists?` is deprecated in favor of `File.exist?`.


<!-- Comment by [Sider](https://sider.review) -->